keylime/cmd/create_policy.py

```diff
@@ -584,6 +585,9 @@ def main() -> None:
     policy["excludes"] = sorted(list(set(policy["excludes"])))
     policy["ima"]["ignored_keyrings"] = sorted(list(set(policy["ima"]["ignored_keyrings"])))
 
+    policy["meta"]["generator"] = ima.RUNTIME_POLICY_GENERATOR.LegacyAllowList
```

Comment:
What is the meaning of `LegacyAllowList`?

All generated runtime policies will be unconditionally be marked as `LegacyAllowList`, is this correct?

Reply:
Yes. I don't know what `LegacyAllowList` means, nor the meaning of the alternative `CompatibelAllowList`.

Reply:
@maugustosilva You introduced this on c6d3d0b

What should be the value set here?

Reply:
The main goal on the introduction of a `generator` attribute was to allow the `verifier` to be able to differentiate between a scenario where an `agent` was added (via `keylime_tenant`) with vs without `--allowlist`. 

Since this option is no longer valid, and now all policies are automatically converted to the new format, we could switch to `CompatibleAllowList`, although I don't think it has to be done on this PR. The `generator` field only has effect on deciding the value of `has_runtime_policy` attribute, and not on the actual processing of the policy and `agent` attestation.